salt/modules/composeng.py

```diff
@@ -0,0 +1,523 @@
+# -*- coding: utf-8 -*-
+'''
+Work with containers managed by docker-compose
+
+Docker-compose method supported
+-------------------------------
+- up
+- restart
+- stop
+- start
+- pause
+- unpause
+- kill
+- rm
+- ps
+- pull
+- build
+'''
+
+from __future__ import absolute_import
+import io
+import logging
+import sys
+import traceback
+import os
+import salt.utils
+from collections import namedtuple
+from contextlib import contextmanager
+from operator import attrgetter
+
+import yaml
+from docopt import docopt
```

Comment:
please do `import docopt` and refer to it as `docopt.docopt` otherwise, `composeng.docopt` will get added to the functions in the loader.